.orchestra/config/components/revng_container.yml

```diff
@@ -0,0 +1,26 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_container():
+binary_archives: private
+configure: |
+  mkdir -p "$BUILD_DIR" && cd "$BUILD_DIR"
+  touch empty_directory_ignore_it
```

Comment:
Is this needed?

Reply:
I copied it from `revng_distributable.yml`, dunno if it's needed

Reply:
Drop drop drop